src/main/java/app/bpartners/api/endpoint/event/model/UserRegistrationRequested.java

```diff
@@ -0,0 +1,31 @@
+package app.bpartners.api.endpoint.event.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.time.Duration;
+import lombok.*;
+
+@Builder
+@ToString
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class UserRegistrationRequested extends PojaEvent {
+  @JsonProperty("userId")
+  private String userId;
+
+  @JsonProperty("userNb")
+  private int userNb;
+
+  @JsonProperty("totalNbUser")
+  private int totalNbUser;
+
+  @Override
+  public Duration maxConsumerDuration() {
+    return Duration.ofMinutes(2);
```

Comment:
Why two ? Why not one nor five ?

Reply:
For the first call, we'll have to register several users, so it'll take a little longer, then for the other calls, we'll probably only register a few users.